src/diffusers/pipelines/sana_video/pipeline_sana_video_i2v.py

```diff
@@ -377,10 +376,10 @@ def encode_prompt(
 
         bs_embed, seq_len, _ = prompt_embeds.shape
         # duplicate text embeddings and attention mask for each generation per prompt, using mps friendly method
-        prompt_embeds = prompt_embeds.repeat(1, num_videos_per_prompt, 1)
-        prompt_embeds = prompt_embeds.view(bs_embed * num_videos_per_prompt, seq_len, -1)
+        prompt_embeds = prompt_embeds.repeat(1, num_images_per_prompt, 1)
+        prompt_embeds = prompt_embeds.view(bs_embed * num_images_per_prompt, seq_len, -1)
         prompt_attention_mask = prompt_attention_mask.view(bs_embed, -1)
-        prompt_attention_mask = prompt_attention_mask.repeat(num_videos_per_prompt, 1)
+        prompt_attention_mask = prompt_attention_mask.repeat(num_images_per_prompt, 1)
```

Comment:
```suggestion
        prompt_attention_mask = prompt_attention_mask.repeat(num_videos_per_prompt, 1)
```
See https://github.com/huggingface/diffusers/pull/12634#discussion_r2529398074